specs/trust-store-trust-policy.md

```diff
@@ -88,7 +128,70 @@ Trust policy for a simple scenario where ACME Rockets uses only artifacts signed
 }
 ```
 
-Trust policy for the scenario where ACME Rockets uses some artifacts signed by Wabbit Networks and some signed by ACME Rockets.
+2.Trust policy for a scenario where ACME Rockets consumes some OCI artifacts signed by Wabbit Networks and some signed by ACME Rockets.
+
+```jsonc
+{
+    "version": "1.1",
+    "trustPolicies": [
+        {
+            // Policy for set of artifacts signed by Wabbit Networks
+            // that are pulled from ACME Rockets repository
+            "name": "wabbit-networks-images",
+            "scopes": [
+              "oci:registry.acme-rockets.io/software/net-monitor",
+              "oci:registry.acme-rockets.io/software/net-logger"
+            ],
+            "signatureVerification": {
+              "level" : "strict" 
+            },
+            "trustStores": ["wabbit-networks"],
+            "trustedIdentities": [ 
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Security Tools"
+            ]
+        },
+        {
+            // Exception policy for a single unsigned OCI artifact pulled from
+            // Wabbit Networks repository
+            "name": "unsigned-image",
+            "scopes": [ "oci:registry.wabbit-networks.io/software/unsigned/net-utils" ],
+            "signatureVerification": {
+              "level" : "skip" 
+            }
+        },
+        {
+            // Policy that uses custom verification level to relax the strict verification.
+            // It logs expiry and skips revocation check for a specific OCI artifact.
+            "name": "use-expired-image",
+            "scopes": [ "oci:registry.acme-rockets.io/software/legacy/metrics" ],
+            "signatureVerification": {
+              "level" : "strict",
+              "override" : {
+                "expiry" : "log",
+                "revocation" : "skip"
+              }
+            },
+            "trustStores": ["ca:acme-rockets"],
+            "trustedIdentities": ["*"]
+        },
+        {
+            // Policy for all other OCI artifacts signed by ACME Rockets
+            // from any registry location. The policy also specified multiple trust stores.
+            "name": "global-policy-for-all-other-images",
+            "scopes": [ "oci:*" ],
+            "signatureVerification": {
+              "level" : "audit"
+            },
+            "trustStores": ["ca:acme-rockets", "ca:acme-rockets-ca2"],
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io, OU=Finance, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+3.Trust policy in version 1.0 for a scenario where ACME Rockets consumes some OCI artifacts signed by Wabbit Networks and some signed by ACME Rockets.
```

Comment:
I'd change 1.0 policy examples as a sub heading so that it's easy to find, and retain all the previous examples in 1.0